--------------------------------------------------------------------------
[[27609,1],0]: A high-performance Open MPI point-to-point messaging module
was unable to find any relevant network interfaces:

Module: OpenFabrics (openib)
  Host: GCRAZGDL0429

Another transport will be used instead, although this may result in
lower performance.

NOTE: You can disable this warning by setting the MCA parameter
btl_base_warn_component_unused to 0.
--------------------------------------------------------------------------
Warning: Log dir exp/wtrpo_CartPole-v1/wtrpo_CartPole-v1_s3 already exists! Storing info there anyway.
[32;1mLogging data to exp/wtrpo_CartPole-v1/wtrpo_CartPole-v1_s3/progress.txt[0m
[36;1mSaving config:
[0m
{
    "ac_kwargs":	{
        "hidden_sizes":	[
            64,
            64
        ]
    },
    "actor_critic":	"mlp_actor_critic",
    "algo":	"trpo",
    "backtrack_coeff":	0.8,
    "backtrack_iters":	10,
    "cg_iters":	10,
    "damping_coeff":	0.1,
    "delta":	0.01,
    "env_fn":	"<function <lambda> at 0x7f3aa6222290>",
    "epochs":	50,
    "exp_name":	"wtrpo_CartPole-v1",
    "gamma":	0.99,
    "lam":	0.97,
    "logger":	{
        "<spinup.utils.logx.EpochLogger object at 0x7f3aa623ddd0>":	{
            "epoch_dict":	{},
            "exp_name":	"wtrpo_CartPole-v1",
            "first_row":	true,
            "log_current_row":	{},
            "log_headers":	[],
            "output_dir":	"exp/wtrpo_CartPole-v1/wtrpo_CartPole-v1_s3",
            "output_file":	{
                "<_io.TextIOWrapper name='exp/wtrpo_CartPole-v1/wtrpo_CartPole-v1_s3/progress.txt' mode='w' encoding='UTF-8'>":	{
                    "mode":	"w"
                }
            }
        }
    },
    "logger_kwargs":	{
        "exp_name":	"wtrpo_CartPole-v1",
        "output_dir":	"exp/wtrpo_CartPole-v1/wtrpo_CartPole-v1_s3"
    },
    "max_ep_len":	1000,
    "save_freq":	10,
    "seed":	3,
    "steps_per_epoch":	4000,
    "train_v_iters":	80,
    "vf_lr":	0.001
}
Traceback (most recent call last):
  File "wtrpo.py", line 404, in <module>
    logger_kwargs=logger_kwargs)
  File "wtrpo.py", line 218, in wtrpo
    pi, logp, logp_pi, info, info_phs, d_wasserstein, v = actor_critic(x_ph, a_ph, **ac_kwargs)
  File "/home/v-tongcheng/Projects/spinningup/spinup/algos/tf1/wtrpo/core.py", line 175, in mlp_actor_critic
    policy_outs = policy(x, a, hidden_sizes, activation, output_activation, action_space)
  File "/home/v-tongcheng/Projects/spinningup/spinup/algos/tf1/wtrpo/core.py", line 135, in mlp_categorical_policy
    d_wasserstein = categorical_wasserstein(logp_all, old_logp_all)
  File "/home/v-tongcheng/Projects/spinningup/spinup/algos/tf1/wtrpo/core.py", line 83, in categorical_wasserstein
    w_dist = wasserstein_distance(tf.exp(logp0), tf.exp(logp1))
  File "/anaconda/envs/rl/lib/python3.7/site-packages/scipy/stats/stats.py", line 8319, in wasserstein_distance
    return _cdf_distance(1, u_values, v_values, u_weights, v_weights)
  File "/anaconda/envs/rl/lib/python3.7/site-packages/scipy/stats/stats.py", line 8448, in _cdf_distance
    u_values, u_weights = _validate_distribution(u_values, u_weights)
  File "/anaconda/envs/rl/lib/python3.7/site-packages/scipy/stats/stats.py", line 8512, in _validate_distribution
    values = np.asarray(values, dtype=float)
TypeError: __array__() takes 1 positional argument but 2 were given
